cmd/parse-genesis/parseGenesis.go

```diff
@@ -45,6 +47,10 @@ func NewParseGenesisCmd(parseCfg *parse.Config) *cobra.Command {
 				return fmt.Errorf("error while getting genesis doc and state: %s", err)
 			}
 
+			if err := resolveInitialHeightBlock(parseCtx, genesisDoc.InitialHeight); err != nil {
```

Comment:
check if block already exists, it it does - skip this function

Reply:
Fixed